pkg/bundler/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@entropyxyz/x25519-chacha20poly1305-bundler",
```

Comment:
I wasn't able to generate a package.json with the name and appended `-<platform>`, ended up adding them manually.

Reply:
additionally, looked into the docs to see if we could generate all targets in one npm module, but didn't look possible.

I don't have any opinions one having a package for each target or one package with different directories for each target.

Starting to think the latter makes sense and we could write a bash script or something to automated this.

wdyt?

Reply:
Hm, could you link towards the docs so I may have a read?

> I don't have any opinions one having a package for each target or one package with different directories for each target.

Probably a package for each target? I'm not sure though, we could look at how Mach-O fat binaries contain both x86 and arm instructions, choosing the appropriate assembly for the processer on hand. Certainly could write a quick bash script for this though.